cloud-controller-manager/do/loadbalancers.go

```diff
@@ -431,57 +433,73 @@ func buildForwardingRules(service *v1.Service) ([]godo.ForwardingRule, error) {
 	if err != nil {
 		return nil, err
 	}
-
-	certificateID := getCertificateID(service)
-	tlsPassThrough := getTLSPassThrough(service)
-
-	if (certificateID != "" || tlsPassThrough) && len(tlsPorts) == 0 {
+	if len(tlsPorts) == 0 {
```

Comment:
Strictly speaking, aren't we diverging from the current behavior with this change?

Before, it was possible for users to run TCP or HTTP traffic on port 443 if they did not specify a certificate ID or TLS passthrough. We now seem to assume that port 443 always means HTTPS.

I realize it'd be somewhat weird to run no-TLS traffic on port 443 (maybe except for testing purposes). If it's possible to implement with the vanilla DO LB product, however, then I believe we should continue to allow so as well with our Kubernetes integration.

WDYT?

Reply:
Definitely agree, the behaviour should remain the same, will update :+1:

Reply:
We may also want to add a test for this case as we are seemingly missing it.